### ISSUE TITLE
ENH: Pinnable tooltips, to allow copying

### ISSUE THREAD

**Issue by corneliusroemer** [nextstrain/nextclade#734]
It would be nice for Nextclade web users to be able to copy mutation profiles etc. directly from the UI.

Right now, this is possible only with some skill or luck, as one needs to move the mouse quickly enough onto the tooltip so that it doesn't disappear.

It would be more accessible, if a click on the area where the tooltip is activate would fix the toop, with a click outside of the tooltip (and maybe an `x` at the top right) deactivating the fixation.

I think I remember @trvrb mentioning being interested in this a while ago.

**Comment by ivan-aksamentov** [comment]
I agree. The tooltips in general is not a great user experience. I considered implementing something better for some time now: https://github.com/nextstrain/nextclade/issues/83 https://github.com/nextstrain/nextclade/issues/84 https://github.com/nextstrain/nextclade/issues/85

**Comment by ivan-aksamentov** [comment]
> if a click on the area where the tooltip is activate would fix the toop, with a click outside of the tooltip (and maybe an x at the top right) deactivating the fixation

This will probably mess up all the touch screens. Might work given some effort though. But technically it would be just easier to have a normally positioned, conventional popup window, like the one with changelog. Could also fit a lot more data and be scrollable.

**Comment by corneliusroemer** [comment]
What about adding a delay on purpose during which time one can move the mouse over the tooltip to fix it in place until the mouse moves away? This would not degrade touch experience

**Comment by emmahodcroft** [comment]
Just as a workaround, I found (at least on Windows 10 chrome) if you get the pop-up up, right-click the mutation, then move your mouse over the pop-up, left-click to dismiss the 'right-click-menu' then you can move the mouse around within the popup, copy text, etc.
Here's a gif:
![pop-up-sticky](https://user-images.githubusercontent.com/14290674/157639782-729b6510-ed6a-4578-b971-94f1bf663b69.gif)
 
 (Sorry I don't know why the colours are weird - that's definitely from making the gif, not how the Nextclade page is actually showing up for me.)

**Comment by ivan-aksamentov** [comment]
I made a little prototype https://github.com/nextstrain/nextclade/pull/749
